jobs/ci-run/integration/gen/test-user.yml

```diff
@@ -32,16 +32,128 @@
     - multijob:
         name: 'IntegrationTests-user'
         projects:
-        - name: 'test-user-lxd'
+        - name: 'test-user-test-user-login-password-lxd'
           current-parameters: true
-        - name: 'test-user-aws'
+        - name: 'test-user-test-user-login-password-aws'
           current-parameters: true
+        - name: 'test-user-test-user-manage-lxd'
+          current-parameters: true
+        - name: 'test-user-test-user-manage-aws'
+          current-parameters: true
+
+- job:
+    name: test-user-test-user-login-password-lxd
+    node: ephemeral-focal-8c-32g-amd64
```

Comment:
Can this use `ephemeral-focal-small-amd64`?

Reply:
Again we use the template for code generation. So, to manage these issues, we need to change the logic for example create additional config files with test-lists for "different" node types.